packages/manager/modules/pci/src/projects/project/data-processing/submit-job/translations/Messages_fr_FR.json

```diff
@@ -7,6 +7,9 @@
   "data_processing_submit_job_stepper_job_type_title": "Sélectionner un type de job",
   "data_processing_submit_job_stepper_sizing_title": "Dimensionnement du job",
   "data_processing_submit_job_stepper_configure_title": "Configurer votre job",
+  "data_processing_submit_job_stepper_submit_title": "Soumettre votre job",
+  "data_processing_submit_job_command_description": "La commande suivante vous permet de créer votre job grâce à",
+  "data_processing_submit_job_command_api_link": "notre API",
```

Comment:
This content should not be splitted. The whole sentence should be a link instead, to avoid splitting translation

Reply:
Should be better now